fashion-example-query/app.py

```diff
@@ -61,7 +49,7 @@ def print_result(resp):
         result_html.append(f'<tr><td><img src="{vi}"/></td><td>')
         for kk in d.matches:
```

Comment:
I think the variables within print_result() is not very readable. e.g. Shall we change d, vi, kk to more readable names?

Reply:
if u have a way to do so, u can also do that in the `hello-world` in the core for sure

Reply:
I'm not sure what d, vi, kk, kmi represent. I would prefer some more descriptive ones, but not sure how to rename them.
And I also feel like 'd' is not a good naming convention. For example, if we use pdb to debug here, 'd' won't work. (https://stackoverflow.com/questions/15160946/python-debugger-tells-me-value-of-numpy-array-is-newest-frame).
What do you think?

Reply:
vi is `uri` and kk is `match`

Reply:
Yeah. I mean what are the full names of these abbreviations. 😄  
My thought is 
- d->data or query_data
- vi->data_uri or query_data_uri
- kk->match
- kmi->match_uri

Reply:
Good lets go

Reply:
Sounds good, so we change it in this example and in the hello-workd example in core right?